assets/src/js/country_selector.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * The script basically add/remove the tabindex attribute within the country selector
+ * It helps to improve SEO
+ * @return {void}
+ */
+export const setupCountrySelector = () => {
+  const targetNode = document.querySelector('#country-selector');
+  const countriesList = document.querySelector('.countries-list');
+  const footerMenu = document.querySelector('.footer-menu');
+  const footerSocialMedia = document.querySelector('.footer-social-media');
+
+  if (!targetNode || !countriesList || !footerMenu || !footerSocialMedia) {
+    return;
+  }
+
+  const largeAndUpScreens = '(min-width: 992px)';
+  const countriesListTransitionDuration = (
+    parseFloat(getComputedStyle(document.querySelector('.countries-list')).transitionDuration) * 1000
+  ) || 500;
+
+  /**
+   * This funtion re-order the elements from the footer
+   * @param {*} evt
+   */
+  const switchOrders = (evt = null) => {
+    const event = evt ? evt : window.matchMedia(largeAndUpScreens);
+    if (event.matches) {
+      footerMenu.parentNode.insertBefore(footerMenu, footerMenu.parentNode.firstChild);
+    } else {
+      footerSocialMedia.parentNode.insertBefore(footerSocialMedia, footerSocialMedia.parentNode.firstChild);
```

Comment:
Maybe this way is easier to read?
```suggestion
      footerSocialMedia.parentNode.prepend(footerSocialMedia);
```